docs/source/contributing.rst

```diff
@@ -24,7 +24,7 @@ Before sending us a Pull Request, please consider this:
 * Are comments and documentation written in clear English, with no spelling or grammar errors?
 
 * Example contributions are also welcome.
-  * If you are contributing by adding a new example, please use the `Arduino style guide`_.
+  * If you are contributing by adding a new example, please use the `Arduino style guide`_ and the example guideline below.
```

Comment:
Missing empty line between:
`* Example contributions are also welcome.`
and
`* If you are contributing by adding a new example, please use the...`